The job **`tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/16242876747/job/45861628876) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/45861628876) [(plain)](https://github.com/rust-lang/rust/commit/69b9bae57ded527f5fd7f036b6cc1cca0b462ac4/checks/45861628876/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
 
 /// The internal representation of a `Thread` handle
-/// 
+///
 /// We explicitly set the alignment for our guarantee in Thread::into_raw. This
 /// allows applications to stuff extra metadata bits into the alignment, which
 /// can be rather useful when working with atomics.
fmt: checked 6159 files
Build completed unsuccessfully in 0:00:43
  local time: Sat Jul 12 23:09:04 UTC 2025
  network time: Sat, 12 Jul 2025 23:09:04 GMT
##[error]Process completed with exit code 1.
```

</details>

--- a/library/std/src/thread/mod.rs
+++ b/library/std/src/thread/mod.rs
@@ -1380,7 +1380,7 @@ where
 }
 
 /// The internal representation of a `Thread` handle
-/// 
+///
 /// We explicitly set the alignment for our guarantee in Thread::into_raw. This
 /// allows applications to stuff extra metadata bits into the alignment, which
 /// can be rather useful when working with atomics.

